src/t8_forest/t8_forest_cxx.cxx

```diff
@@ -1278,29 +1352,43 @@ t8_forest_element_point_inside (t8_forest_t forest, t8_locidx_t ltreeid, const t
      *  <x - p, n> >= 0
```

Comment:
I cannot comment on line 1346 directly, but the formatting is weird in that comment

Reply:
Updated the format.